src/NTZ/README.md

```
@@ -0,0 +1,51 @@
+
+**Verification**:
+You can verify the certificate details yourself using the included intermediate certificate:
+```bash
+openssl x509 -in src/NTZ/EkMfrCA001.crt -noout -text | grep -A2 "Authority Information Access"
+```
+Expected output should contain: `CA Issuers - URI:http://pki.nationz.com.cn/EkMfrCA001/EkMfrCA001.crt`
+
+#### Source Information
+
+- **Screenshot Reference**: 
+![](h3c_support.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)